### PR TITLE
uniqueId property

### PR DESCRIPTION
**Pull request by padawannn**

### CODE REVIEW

File: docs/api-reference/carto/carto-bqtiler-layer.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ new deck.carto.CartoBQTilerLayer({});
 
 ## Properties
 
+##### `uniqueIdProperty` (String | Number)
```
Comment (VictorVelarde) [inline docs/api-reference/carto/carto-bqtiler-layer.md:59]: Isn't this property always a 'string'? --> the 'name' of the field?

File: docs/api-reference/carto/carto-bqtiler-layer.md [hunk 2]
```diff
@@ -56,6 +56,11 @@ new deck.carto.CartoBQTilerLayer({});
...
+
+
+Optional. An string or number pointing to a tile attribute containing a unique identifier for features across tiles. If not, [feature id](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#42-features) will be used.
```
Comment (VictorVelarde) [inline docs/api-reference/carto/carto-bqtiler-layer.md:62]: Should be... "A string pointing to a tile attribute, containing a number or a string as a unique identifier...

**Review by alasarr — APPROVED** [review]

File: docs/api-reference/carto/carto-bqtiler-layer.md [hunk 3]
```diff
@@ -56,11 +56,18 @@ new deck.carto.CartoBQTilerLayer({});
...
 
 Required. Tileset id
 
+##### `uniqueIdProperty` (String)
+
+* Default: `id`
+
+Optional. Needed for highlighting a feature split across two or more tiles if no [feature id](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#42-features) is provided.
+
+An string pointing to a tile attribute containing a unique identifier for features across tiles.
```
Comment (VictorVelarde) [inline docs/api-reference/carto/carto-bqtiler-layer.md:69]: typo --> 'a~n~ string'

File: docs/api-reference/carto/carto-sql-layer.md [hunk 4]
```diff
@@ -58,11 +58,17 @@ new deck.carto.CartoSQLLayer({});
...
-
 ##### `data` (String)
 
 Required. Either a sql query or a name of dataset
 
+##### `uniqueIdProperty` (String)
+
+* Default: `cartodb_id`
+
+Optional. An string pointing to a unique attribute at the result of the query. An unique attribute is needed for highlighting a feature split across two or more tiles.
```
Comment (VictorVelarde) [inline docs/api-reference/carto/carto-sql-layer.md:69]: same typo

Comment (VictorVelarde) [reply docs/api-reference/carto/carto-sql-layer.md:69]: Also 'a~n~ unique'

Comment (alasarr) [reply docs/api-reference/carto/carto-sql-layer.md]: Done!

**Review by alasarr — APPROVED** [review]

**Review by VictorVelarde — APPROVED** [review]
👍

**Pull request merged**